cardano-node/src/Cardano/CLI/Ops.hs

```diff
@@ -350,19 +351,19 @@ withRealPBFT
         -> ExceptT RealPBFTError IO a)
   -> ExceptT RealPBFTError IO a
 withRealPBFT genFile nMagic sigThresh delCertFp sKeyFp update ptcl action = do
-  SomeProtocol p <- firstExceptT
-                      FromProtocolError
-                      $ fromProtocol
-                          ptcl
-                          Nothing
-                          Nothing
-                          (Just genFile)
-                          nMagic
-                          sigThresh
-                          delCertFp
-                          sKeyFp
-                          update
-  case p of
+  SomeConsensusProtocol cp <-
```

Comment:
Can we avoid making semantically null whitespace changes? Changes that include whitespace and non-obvious changes in the same PR are more difficult to review.

Reply:
done